invoke.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
```

Comment:
This file probably doesn't belong in here.  I wrote this as a quick hack to extract the java/lang/invoke API from metadata.  Even then, there were plenty of manual fixup steps after running it, so it's not readily reusable.